extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -895,6 +904,28 @@ export class AmpAutocomplete extends AMP.BaseElement {
     }
   }
 
+  /**
+   * If a fallback element is provided, displays it instead of suggestions.
+   * Otherwise, throws given error. Must be called in a mutate context.
+   * @param {*=} error
+   * @throws {!Error} If fallback element is not present.
```

Comment:
nit: A `user().warn` isn't the same as throwing an error, it's only like calling `console.warn()`